analysis_options.yaml

```diff
@@ -1 +1,7 @@
+analyzer:
+  language:
+    strict-casts: true
+    strict-inference: true
+    strict-raw-types: true
```

Comment:
is this not included by flame_lint???
I am in shock

Reply:
I really thought it was too